.settings/org.eclipse.core.resources.prefs

```diff
@@ -25,16 +25,16 @@ encoding//Debug/ex2_hal/adcs/equipment_handler/src/subdir_rules.mk=UTF-8
 encoding//Debug/ex2_hal/adcs/equipment_handler/src/subdir_vars.mk=UTF-8

```

Comment:
COuld you remove this file? It never should have been added to the index from the start